app/src/main/java/com/example/mymovieinfo/ui/theme/TextType.kt

```diff
@@ -0,0 +1,16 @@
+package com.example.mymovieinfo.ui.theme
+
+import androidx.compose.material.Typography
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.sp
+
+val Typography = Typography(
+    body1 =  TextStyle(
+        fontFamily = FontFamily.SansSerif,
+        fontWeight = FontWeight.Normal,
+        fontSize = 16.sp
+//add back up set of typography just in case
```

Comment:
It looks like this group is using XML layouts.  If that's the case, Compose themes will not be used, and this can be discarded.